### PR TITLE
feat(blog): add day in the life blog post

### PR DESCRIPTION
**Pull request by DSchau**
## Description

This WIP blog post outlines the day-to-day responsibilities and tasks of a Core maintainer. It feels like it's in pretty good shape, but thoughts and insights more than welcome!

localhost_8000_blog_2019-06-17-day-in-the-life-of-a-core-maintainer

### CODE REVIEW

File: docs/blog/2019-06-17-day-in-the-life-of-a-core-maintainer/index.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+- **Helpful, targeted use of expertise**. Unblocking a community member or Gatsby team member, which may take just a few minutes of your time may unblock and drastically reduce the time spent on a particularly tough issue by an ally. If someone surfaces a question in - Slack or Discord—if you have a few moments, help them out!
+- **Pairing**. Leveraging your skill set to help another member of the team is a great way to - get a meaningful feature delivered more quickly. Hop on a Zoom and build something great!
+- **Writing**. Writing docs, writing a blog post, speaking at a conference or on a podcast, - cultivating community members to level-up, etc. [See “Why we Write” by Sam Bhagwat](https://www.gatsbyjs.org/blog/2019-04-19-gatsby-why-we-write/).
+- **Creating issues**. Create and/or edit issues, RFCs, etc. for community members to discuss or pick up. If we can let other people work on relatively simpler tasks, it frees us up to work on higher-value tasks, which tend to be more complex and require more focused - effort.
```
Comment (pieh) [inline docs/blog/2019-06-17-day-in-the-life-of-a-core-maintainer/index.md]: > work on relatively simpler tasks
I don't love this phrase, but I also don't have suggestion on how to rephrase that. 

Maybe something more in the lines of "If we can get help from the community with shipping features, fixing bugs, and improving documentation, it frees us up to work on large-scale projects"?

File: docs/blog/2019-06-17-day-in-the-life-of-a-core-maintainer/index.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+- the community
+- Gatsby Core, or
+- other teams at Gatsby.
+
+An ideal scenario is work that intersects with all three, or that unblocks or leads to future work that can reach a wider audience.
+
+Some examples:
+
+- **Shipping roadmap features**. Many Roadmap features are usually high-effort but often - hugely impactful in helping large numbers of Gatsby users.
+- **Shipping meaningful, small bug-fixes**. Many small bug-fixes are low-effort and medium-impact. They "move the needle" and improve the day-to-day experience of using - Gatsby for a non-trivial amount of users.
```
[the remark this answers is not in the document]
Comment (DSchau) [reply docs/blog/2019-06-17-day-in-the-life-of-a-core-maintainer/index.md]: 100% -- was a typo!

File: docs/blog/2019-06-17-day-in-the-life-of-a-core-maintainer/index.md (hunk 1 above)
Comment (DSchau) [reply docs/blog/2019-06-17-day-in-the-life-of-a-core-maintainer/index.md]: ^ way better way of saying it. I'll address.

File: docs/blog/2019-06-17-day-in-the-life-of-a-core-maintainer/index.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+
+- the community
+- Gatsby Core, or
+- other teams at Gatsby.
+
+An ideal scenario is work that intersects with all three, or that unblocks or leads to future work that can reach a wider audience.
+
+Some examples:
+
+- **Shipping roadmap features**. Many Roadmap features are usually high-effort but often - hugely impactful in helping large numbers of Gatsby users.
```
Comment (gillkyle) [inline docs/blog/2019-06-17-day-in-the-life-of-a-core-maintainer/index.md]:
```suggestion
- **Shipping roadmap features**. Many roadmap features are usually high-effort, but often hugely impactful in helping large numbers of Gatsby users.
```

File: docs/blog/2019-06-17-day-in-the-life-of-a-core-maintainer/index.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+
+## Middle of day to end of day (4-6 hours)
+
+Feeling pretty good! Community focused tasks make it clear that the community is, and will always be, a priority for Gatsby. However--we also want to measurably move the needle forward and make Gatsby better for everyone. This will often involve features and fixes that require a deep context and understanding of Gatsby internals. Oftentimes (perhaps most of the time) these will correspond to a roadmap item which will support eventual larger goals tying to an Objective and Key Result (OKR), Key Product Initiative (KPI), and/or another team at Gatsby, e.g. Cloud. These are features that require deliberate focus to think about, approach, and ship with minimal outside distractions. To this end, some have found it valuable to block off chunks of time in their calendar so that meetings can't be scheduled, Slack is in do not disturb, etc. If these measures help you to truly focus and ship impactful features, by all means--do so! All at Gatsby recognize the value of focus and how it can effectively drive key results and features.
```
Comment (gillkyle) [inline docs/blog/2019-06-17-day-in-the-life-of-a-core-maintainer/index.md]: I had thought KPI stood for Key Performance Indicator, though both sound pretty similar. I could be wrong 🤔

Comment (calcsam) [reply docs/blog/2019-06-17-day-in-the-life-of-a-core-maintainer/index.md]: @gillkyle is correct :)

Comment (DSchau) [reply docs/blog/2019-06-17-day-in-the-life-of-a-core-maintainer/index.md]: This is the problem with acronyms 😅 will fix! Thanks Kyle and Sam!

Comment (DSchau) [reply docs/blog/2019-06-17-day-in-the-life-of-a-core-maintainer/index.md]: Fixed :)

**Pull request merged**